src/lib/seam/use-seam-client.ts

```diff
@@ -52,34 +55,24 @@ export function useSeamClient(): {
   return { client: data ?? null, isLoading, isError, error }
 }
 
-function useClientSession(): [
-  clientSession: ClientSession | null,
-  setClientSession: (clientSession: ClientSession) => void
-] {
-  const localStorageKey = 'seam_client_session'
-
-  const setClientSession = (clientSession: ClientSession): void => {
-    globalThis?.localStorage?.setItem(
-      localStorageKey,
-      JSON.stringify(clientSession)
-    )
-  }
-
-  const cachedClientSession = globalThis?.localStorage?.getItem(localStorageKey)
-
-  return [
-    cachedClientSession != null ? JSON.parse(cachedClientSession) : null,
-    setClientSession,
-  ]
-}
-
-function useUserIdentifierKey(userIdentifierKey: string | undefined): string {
+/**
```

Comment:
Cool. Better than a comment IMO: print a warning to console as well so the devs will see this!